### PR TITLE
Oshift review lifecycle patch

### PR DESCRIPTION
**Pull request by koneill94**
Updated to be relevant for v4. Needs our other outstanding branch around upgrades to be merged so I can provide a link to that in here (placeholder entered currently) but please review the content apart from that so any necessary changes can be made.

### CODE REVIEW

**Review by suehighmoor — APPROVED** [review]
LGTM

**Comment by suehighmoor** [comment]
PR1612 has been merged, so the link should work when this article is published.

File: articles/openshift/oshift-ref-lifecycle.md [hunk 1]
```diff
@@ -29,22 +29,19 @@ Administrators and operators of the UKCloud for Managed OpenShift service.
...
+Currently, UKCloud does not offer in-place upgrades between major versions. Customers can trigger minor version upgrades inside the cluster at any time, providing total flexibility to upgrade at a convenient time.
 
-To facilitate any upgrades, UKCloud will deploy a new environment to customers based on the latest tested and approved release of OpenShift free of charge for thirty days in order to conduct any testing and migrate workloads over to the new environment. At the end of thirty days, UKCloud will shut down and remove the old environment to prevent any double-charging.
-
-> [!NOTE]
-> UKCloud will actively work with you during this period to help ensure a smooth migration and will not turn off any old environments until you advise us that it's safe to do so.
+We require customers to stay on UKCloud-supported versions, so ask that upgrades are done in a timely manner to stay on a Y-1 release. For more information on performing updates and what happens if customers aren't on Y-1, see [*Supported customer update strategy for version 4 clusters*](oshift-how-v4-update.md). <need update branch merged first to add URL to these docs>
```
Comment (agarthetiger) [inline articles/openshift/oshift-ref-lifecycle.md:40]: Presumably "<need update branch merged first to add URL to these docs>" should be removed now before we merge this branch?

**Review by agarthetiger — CHANGES_REQUESTED** [review]
There is also diagram on this page which still depicts that we support N-2 minor versions, we should change this diagram too, maybe create a new one if we don't have the source for the original (looks like it was done in Excel to me).

File: articles/openshift/oshift-ref-lifecycle.md (hunk 1 above)
Comment (suehighmoor) [reply articles/openshift/oshift-ref-lifecycle.md]: Thanks for spotting that. Annoyed with myself that I left it in :(

**Comment by suehighmoor** [comment]
@agarthetiger - the image was updated, but maybe didn't show up in the view you were using. Updated image is here:
https://github.com/UKCloud/documentation/blob/d48e02f0515684d83cb227ff6dcd95fc46191c30/articles/openshift/images/oshift-supported-versions.png

**Review by benbacon — APPROVED** [review]
Looks good to me

**Review by agarthetiger — APPROVED** [review]
All looks good to me now. Thanks :)

**Pull request merged**